examples/check-if-opened-with-testcafe/README.MD

```diff
@@ -0,0 +1,17 @@
+# Check Whether Web Page Is Opened With TestCafe
+
+**Test Code**: [index.js](index.js)
+**Tested Page**: [index.html](index.html)
+
+You may need to change how your app operates depending on whether it is currently being tested with TestCafe. This example demonstrates how to let your application know whether it is opened with TestCafe.
```

Comment:
`change how your app operates` sounds like big changes, so it's a bit weird that someone has to make such changes for testing.

Maybe 

*You may need to make adjustments to your application depending on whether it is tested with TestCafe.*

Reply:
I don't think that's quite true. The user wants their app to behave one way if it's launched through TestCafe, and the other if it isn't. There are no adjustments to the app itself- just to the environment. I suggest we leave this as is.

Or:

> You may need to adjust how your app functions depending on whether...

Reply:
edited